data/src/main/java/com/example/pinterbest/data/models/OwnProfile.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.pinterbest.data.models
+
+import com.example.pinterbest.domain.entities.Profile
+import com.google.gson.annotations.SerializedName
+
+data class OwnProfile(
+    @SerializedName("ID")
+    val id: Int,
+    val username: String,
+    val email: String,
+    val avatarLink: String,
+    val following: Int,
+    val followers: Int,
+    val boardsCount: Int,
+    val pinsCount: Int
```

Comment:
remember about pro guard and code obfuscation

Reply:
Fixed